tdp/dao.py

```diff
@@ -120,6 +120,46 @@ def _create_get_sch_latest_status_statement(
     return select(subq).filter(*query_filter)
 
 
+def get_sch_status_history(
+    limit: int,
```

Comment:
You should specify a default value here, eventually None.

Reply:
As an int I cannot put None and the default limit is already defined in the cli

Reply:
Added a `Optional[int]` as type and equal to None. We always define a limit so that the output does not explode in the terminal but this is a display problem which concerns the cli and not a core problem.